treestats_paper.tex

```diff
@@ -931,6 +955,12 @@ \section*{Application to 1000 Genomes tree sequences}
     }
 \end{figure}
 
+\krt{Hmmm...not sure about this paragraph.  Under strict BGS, some of what is stated here doesn't hold.  The tree will
+    still be neutral in shape, just shorter overall.  For the case of many weakly-deleterious segregating mutations, the
```

Comment:
> still be neutral in shape, just shorter overall

Oh, really? I have never heard this, and have seen simulation studies showing that the trees are not the same (e.g. Good et al?). What is "strict BGS"?

Reply:
Strict BGS is as originally defined (CMC '93, etc..), where there is an Ne for the model.    Once you are in that domain, your trees are neutral.  This differs from the colloquial use of BGS, which is often taken to mean any form of purifying selection.  It is important to distinguish the two, as it is easier to get the strong correlation of diversity + recombination rate (w/o increase in divergence) with one and not with the other.

Reply:
Which Good et al are you referring to?

Reply:
How about we just change "background selection" to "linked selection".